template/_templates/graphql/new/graphql.ejs

```diff
@@ -58,7 +58,8 @@ schema.extendType({
 });
 
 // Mutations
-schema.mutationType({
+export const <%= camelized %>Mutations = extendType({
+  type: 'Mutation',
```

Comment:
since we don't have the nexus framework schema singleton, I've moved to named exports